base/BUILD

```diff
@@ -113,73 +113,73 @@ go_binary(
 )
 
 # Replicate the containers and tests for debian9 and debian10
-distro_components("-debian9")
+distro_components("_debian9")
 
-distro_components("-debian10")
+distro_components("_debian10")
 
 # alias debian9 as the default images
 alias(
     name = "static",
-    actual = ":static-debian9",
+    actual = ":static_debian9",
 )
 
 alias(
     name = "static-nonroot",
-    actual = ":static-nonroot-debian9",
+    actual = ":static-nonroot_debian9",
 )
 
 alias(
     name = "base",
-    actual = ":base-debian9",
+    actual = ":base_debian9",
 )
 
 alias(
     name = "base-nonroot",
-    actual = ":base-nonroot-debian9",
+    actual = ":base-nonroot_debian9",
 )
 
 alias(
     name = "debug",
-    actual = ":debug-debian9",
+    actual = ":debug_debian9",
 )
 
 alias(
     name = "debug-nonroot",
-    actual = ":debug-nonroot-debian9",
+    actual = ":debug-nonroot_debian9",
 )
 
 container_test(
-    name = "base_release_test-debian9",
+    name = "base_release_test_debian9",
```

Comment:
Unrelated, but test names usually end with `..._debian9_test`.

Reply:
Nice catch! I tried to correct some of the other ones in this PR so I'll add this one.